site2/docs/functions-debugging.md

```diff
@@ -0,0 +1,121 @@
+---
+id: functions-debugging
+title: Debugging Pulsar Functions
+sidebar_label: Debugging functions
```

Comment:
"Debug Functions" is awkward sounding and confusing as a side part title.  Either we keep this as it is or change it to "How to debug Functions"

Reply:
"Debugging functions" is confusing because users may think "they are functions used with debugging" rather than "How to debug Functions".

Example
Microsoft Debugging Functions: https://docs.microsoft.com/en-us/windows/desktop/debug/debugging-functions.

Why "Debug Functions" is confusing?